ophyd/runengine/runengine.py

```diff
@@ -27,16 +27,15 @@ def _get_info(positioners=None, detectors=None, data=None):
     data : dict
         Dictionary of actual data
     """
-    pv_info = {pos.name: {'source': pos.report['pv'],
-                          'dtype': 'number',
-                          'shape': None}
-               for pos in positioners}
+    src = {}
+    [src.update(x.source) for x in (detectors + positioners)]
 
-    def get_det_info(detector):
+    info_dict = {}
+    for name, value in data.iteritems():
```

Comment:
That documentation is now in a really funny place.  Reading this a bit more closely I think the doc just needs to get removed.